bcda/models/models.go

```diff
@@ -310,7 +309,8 @@ func (aco *ACO) GetNewAndExistingBeneficiaries(includeSuppressed bool, since str
 	// Get all the beneficiaries from the older CCLF for this ACO, to use as a basis for comparison
 	// If the older CCLF was not found, then all beneficiaries will be considered new
 	if foundCclfFileOld {
-		err = db.Table("cclf_beneficiaries").Where("file_id = ?", cclfFileOld.ID).Pluck("mbi", &cclfBeneficiariesOld).Error
+		cclfBeneficiariesOldQuery := fmt.Sprintf("SELECT mbi FROM cclf_beneficiaries WHERE file_id = %d", cclfFileOld.ID)
+		err = db.Raw(cclfBeneficiariesOldQuery).Error
```

Comment:
We don't need this query, right? We're just dropping the results and requerying at lines 329 and 331

Reply:
I was thinking we could "error fast", like we did previously, so as not to affect existing flow.

Reply:
Gotcha, I can see the benefit of the early exit.